config/config.go

```diff
@@ -144,7 +150,7 @@ gui:
 update:
   dockerRefreshInterval: 2000ms`
 
-	ldConfigPath := filepath.Join(ldDirPath, "lazydocker.yml")
+	ldConfigPath := filepath.Join(ldDirPath, "config.yml")
```

Comment:
is it config.yml or lazydocker.yml?

Reply:
`config.yml` https://github.com/jesseduffield/lazydocker/blob/master/docs/Config.md
no clue why `lazydocker.yml` worked, maybe it's an older version that's still backwards compatible?